cmd/start.go

```diff
@@ -63,17 +65,22 @@ var startCmd = &cobra.Command{
 		// read in the sc startup commands
 		file, _ := os.Open(configFile)
 		fscanner := bufio.NewScanner(file)
+		stop := make(chan os.Signal, 1)
 		for fscanner.Scan() {
 			if fscanner.Text() != "" || len(fscanner.Text()) != 0 {
 				wg.Add(1)
-				c := make(chan string)
-				go manager.PoolName(fscanner.Text(), c)
-				pool := <-c
+				metadata := make(chan string)
+				go manager.PoolName(fscanner.Text(), metadata)
+				pool := <-metadata
 				logger.Disklog.Debugf("%s pool is %s.  Metadata is %v", fscanner.Text(), pool, meta[pool])
 				go manager.Start(fscanner.Text(), &wg, meta[pool])
 			}
 		}
-		// TODO catch the ctrl-c here if it comes.
+		signal.Notify(stop, os.Interrupt, syscall.SIGINT, syscall.SIGTERM)
+		go func() {
+			<-stop
+			logger.Disklog.Warn("User pressed ctrl-c. Hard killing tunnels now.  Active jobs using these tunnels will die.")
```

Comment:
Is this true? I started 3 HA tunnels and started a live test. While test was still running, I sent a SIGINT to sauced.
Two tunnels shutdown and the third one waited until the process was finished.

Reply:
now works!  Try shutting down running tunnels with a ctrl-c to sauced.  they will gracefully shutdown